Kernel/KParams.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <AK/AKString.h>
+#include <AK/HashMap.h>
+
+class KParams {
+    AK_MAKE_ETERNAL
+public:
+    static KParams& the();
+
+    KParams(const String cmdline);
+
+    const String cmdline() const;
```

Comment:
There's generally no need to qualify returned-by-value objects with "const" :)
However, this particular function can return a `const String&`.
```suggestion
    const String& cmdline() const { return m_cmdline; }
```